### DISCUSSION TITLE
How to use a list to pass many parameters(such as 100) for iminuit on python

### DISCUSSION THREAD

**Discussion by xiangjuner** [scikit-hep/iminuit#842] [Q&A]
`from iminuit import Minuit`
If i have many parameters(such as 100),my question is how can i use a list to pass parameters and how can I initialize them .From (https://iminuit.readthedocs.io/en/stable/notebooks/basic.html#Initialize-the-Minuit-object) this tutorial ,i can’t find the exact example. The parameters of their examples are too few!

**Comment by HDembinski** [comment]
You can pass parameters as a list, see
https://iminuit.readthedocs.io/en/stable/notebooks/basic.html#Initial-parameter-values
and look for the example with `line_np`. Your model or cost function needs to accept the parameters as a single array. Then you can also pass the parameters to the Minuit object as an array.
